test/acceptance-tests/operator_images_test.go

```diff
@@ -0,0 +1,67 @@
+package acceptance_tests
+
+import (
+	"encoding/json"
+	"fmt"
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	"github.com/securesign/structural-tests/test/support"
+	"log"
+)
+
+var _ = Describe("Trusted Artifact Signer Operator", func() {
+
+	var (
+		snapshotJsonData        support.Snapshot
+		operatorImageDefinition string
+		operatorImages          map[string]string
+	)
+
+	It("get and parse snapshot.json file", func() {
```

Comment:
I would consider providing filepath as an argument to the test suite. We can use local path or `http`.

Reply:
Added possibility to use local path